### ISSUE TITLE
"Governors malformed" custom kernel on Ryzen 7 1700Arch Linux

### ISSUE THREAD

**Issue by CygnusTerminal** [FeralInteractive/gamemode#310]
**Describe the bug**
When running gamemoded -t, the following error appears: ERROR: Governors malformed: got "performance", expected "ondemand"ERROR: Governor was not set to performance (was actually malformed)!
I tried a steam game with gamemoderun command, i checked the governor with cpupower, while the game was running in the background, and after the game was running, it was still "ondemand" and clocks never touched 3.4GHz. The PC is overclocked (3.4GHz) with"core performance boost" disabled and voltage set manually. The module loaded seems to be "acpi-cpufreq"

**To Reproduce**
Steps used to reproduce the behavior:
1. run "gamemoded -t" and observe the error
2. Run any game from steam with the "gamemoderun %command%" option set
3. When game is loaded, minimize and check governor with either "cat /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor" or cpupower-gui
4. Governor still is ondemand

**Expected behavior**
The game runs, and gamemode sets the governor for all cores to "performance"

**System Info (please complete the following information):**
 - OS and version: Arch Linux rolling
 - GameMode Version: 1.6-9

**Additional context**
Neither cpupower nor "echo performance >> /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor" is able to change the governor.

**Comment by poperigby** [comment]
I'm also having this issue with a Ryzen 7 3700X. Did you ever manage to fix this?

**Comment by CygnusTerminal** [comment]
> I'm also having this issue with a Ryzen 7 3700X. Did you ever manage to fix this?

Yes, if i recall correctly it was an issue with corectrl overriding the default governor with ondemand, and thus making gamemode fail.

**Comment by poperigby** [comment]
Ah. Were you able to get gamemode and corectrl to coexist?

**Comment by CygnusTerminal** [comment]
> Ah. Were you able to get gamemode and corectrl to coexist?

Sort of, i had to disable the governor tuning. I switched to radeon-profile shortly after.
